<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-VKNHMMJ" ignored="0" name="Test" tests="2" failures="1" timestamp="23 Feb 2018 13:39:25 GMT" time="0.072" errors="0">
  <testcase name="dum_to_fail" time="0.017" classname="mavpr.mav">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at mavpr.mav.dum_to_fail(mav.java:16)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- dum_to_fail -->
  <testcase name="dummy" time="0.0" classname="mavpr.mav"/>
</testsuite> <!-- Test -->
